external-crates/move/crates/move-compiler/tests/move_check/parser/inferred_string_missing_quote.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/move-compiler/tests/move_check_testsuite.rs
+info:
+  flavor: core
+  edition: legacy
+  lint: false
+---
+error[E01019]: invalid string literal
+  ┌─ tests/move_check/parser/inferred_string_missing_quote.move:3:9
+  │
+3 │         "abcd
```

Comment:
What happens if the string includes unicode characters? Do we handle that correctly? 
(sorry I'm sure this is evident somewhere but its late and am not thinking straight, so maybe I should stop now and check back tomorrow lol)